### PR TITLE
Update infrastructure

### PR DESCRIPTION
**Pull request by tupui**

### CODE REVIEW

**Comment by tupui** [comment]
https://jenkins-os.mapd.com/blue/organizations/jenkins/pyomnisci-tests-pr/detail/pyomnisci-tests-pr/79/pipeline/

File: environment.yml [hunk 1]
```diff
@@ -5,13 +5,11 @@ channels:
 dependencies:
 - pandas
 - pyomniscidb
-- pyarrow==3.0.0
-- thrift<=0.14.0
+- pyarrow>=3.0.0
 - pandas
-- python >=3.7.0
+- python >=3.7.0, <3.10
```
Comment (pearu) [inline environment.yml:10]: What is up with python 3.10?

File: pyproject.toml [hunk 2]
```diff
@@ -1,12 +1,73 @@
...
+    "pyarrow >=3.0.0",
+    "pandas",
+    "numpy",
+    "shapely",
+    "importlib_metadata",  # drop for python >3.7
+]
+
+[project.optional-dependencies]
+gpu = [
+    "numba >=0.49"
```
Comment (pearu) [inline pyproject.toml:30]: I wonder, is this for rbc?

**Review by pearu — APPROVED** [review]
LGTM!

Just wondering, why python 3.10+ is not supported?

**Comment by tupui** [comment]
I am not sure yet about Python 3.10. It was not making the connection and executing the SQL. I will look more on Monday. I can reproduce locally.

**Pull request merged**